Examples/Example-iOS/Source/GTMAppAuthExampleViewController.m

```diff
@@ -107,19 +107,29 @@ - (void)viewDidLoad {
 /*! @brief Saves the @c GTMAppAuthFetcherAuthorization to @c NSUSerDefaults.
  */
 - (void)saveState {
+  NSError *error;
   if (_authorization.canAuthorize) {
     [GTMAppAuthFetcherAuthorization saveAuthorization:_authorization
-                                    toKeychainForName:kExampleAuthorizerKey];
+                                         withItemName:kExampleAuthorizerKey
+                                                error:&error];
```

Comment:
Didn't we decide _not_ to expose the error handling improvements in the public API at this point?

Reply:
Let's talk more about this! I think I'd prefer to leave the error handling in place to help this PR move forward, but I'd be happy to make changes to this API before we land the `swift-translation` branch.